session/src/main/java/org/apache/iotdb/session/pool/SessionPool.java

```diff
@@ -1161,4 +1161,121 @@ public SessionDataSetWrapper executeRawDataQuery(List<String> paths, long startT
     // never go here
     return null;
   }
+
+  public int getMaxSize() {
+    return maxSize;
+  }
+
+  public String getHost() {
+    return host;
+  }
+
+  public int getPort() {
+    return port;
+  }
+
+  public String getUser() {
+    return user;
+  }
+
+  public String getPassword() {
+    return password;
+  }
+
+  public int getFetchSize() {
+    return fetchSize;
+  }
+
+  public long getTimeout() {
+    return timeout;
+  }
+
+  public boolean isEnableCompression() {
+    return enableCompression;
+  }
+
+  public boolean isEnableCacheLeader() {
+    return enableCacheLeader;
+  }
+
+  public ZoneId getZoneId() {
```

Comment:
Currently, the setup of zondId needs a rpc `setTimeZone` which you can refer to [PR3457](https://github.com/apache/iotdb/pull/3457/files), so maybe we can not initialized zoneId here, what do you think?